packages/typespec-ts/test/modularUnit/scenarios/models/template/template.md

```diff
@@ -59,19 +59,19 @@ export function fooSerializer(item: Foo): any {
   return {
     x: templatedBaseSerializer(item["x"]),
     y: templatedSerializer(item["y"]),
-    z: templatedSerializer(item["z"]),
-    h: templatedSerializer(item["h"]),
-    j: templatedSerializer(item["j"]),
+    z: __PLACEHOLDER_o353_sserializer__(item["z"]),
```

Comment:
there's a tcgc issue to track https://github.com/Azure/typespec-azure/issues/2415